src/components/widgets/Map.vue

```diff
@@ -236,6 +265,10 @@ watch(isMouseOver, () => {
 })
 
 onMounted(async () => {
+  widget.value.options.blockGenericContextMenu = true
```

Comment:
This `blockGenericContextMenu` variable is not a widget instance option, as it is not something for the user to configure. It is just a widget type variable.
They way it is right now, it can/will cause saving-loops of the widget profiles.